helm-chart/renku-core/templates/deployment.yaml

```diff
@@ -83,6 +87,10 @@ spec:
               value: {{ .Values.cleanupProjectsTTL | quote }}
             - name: RQ_WORKER_LOG_LEVEL
               value: {{ .Values.logLevel }}
+            - name: SENTRY_DSN
+              value: {{ .Values.sentry.dsn }}
+            - name: SENTRY_ENV
```

Comment:
I think we should think about moving the names of the environment variables to the values file (since they could be considered values as well). Then here we could use `range` to iterate through all `env` entries. 

Something like this:

```
          env:          
            {{- range .Values.env }}
            - name: {{ .name }}
              value: "{{ .value }}"
            {{- end }}
```

and then in values.yaml we could have:

```
env:          
 - name: "SENTRY_DSN"
   value: ""
...
```

Reply:
I could see this being useful for making certain things more flexible, but in general isn't it better/cleaner/less error prone to be more explicit about what is going into the environment? Because in the application the environment variable names are fixed.

Reply:
IMHO, I think it's easier to reason about the helm package when all application-specific defining elements are in 1 place. That way, templates are templates and values contain all application-specific stuff. But maybe that's just me. This is off the scope for this PR anyway, I'm just saying that I think it might be nicer to have all off it in 1 place.

Reply:
I'd say we could discuss this in the context of an overall chart refactoring - would be nice to stay consistent across all the charts we develop. We can open a general issue/epic for this.